### ISSUE TITLE
Scaling option to apply to scale values to print bed vs model

### ISSUE THREAD

**Issue by cwimsatt** [sohjsolwin/M3D#8]
Scaling values currently apply to a model, which is not always desired.  For example, if you want to scale to correct dimensional inaccuracies, then you want the scaling to apply to the print bed axes, regardless of the orientation of the model.  With current scaling, scaling is based on the model's axes.  When you rotate the model 90 degrees about the Z axis, for example, the scaling that you apply to the X axis will change the Y axis relative to the print bed.

**Comment by sohjsolwin** [comment]
If added, I feel this should be available as a toggle-able option through settings, as not every user would like this change. 

One issue with this would be the need to apply an actual transform to the object, rather than just rotating it around various axis. If the object was rotates 45 degrees for example, scaling one axis would not work as expected and would skew the model instead. 

Would adding actual UI handles that can be used for scaling on the object help as well? That way you would be able to grab the specific handle you're attempting to scale, rather than needing to figure out which axis to scale from the side menu.

**Comment by cwimsatt** [comment]
Yes, this would require scaling for two different sets of axes independently.  This would be equivalent to maintaining separate world and object matrices in openGL, yes?  The bed's axes are our analog of the opengl world axes.  We want to scale them to make all prints come out with correct dims.  Perhaps I am thinking of the mechanics of the printer incorrectly, but my assumption has been that the printer has built-in/inherent inaccuracies with it's axes that is constant, regardless of the geometry of the model.  So I always scale my X by .99 and my Y by .98 on my Pro, and my X by .96 and my Y by .99 on my Original Micro (and I don't scale or orient my models with M3D).

This is connected to the #3, so they can be both deferred.  It's a nice to have feature, since there is a relatively painless workaround, namely rotating the model to the desired orientation outside of M3D.
